src/main/java/fr/pickaria/pterodactylpoweraction/configuration/ConfigurationDoctor.java

```diff
@@ -107,7 +107,22 @@ public void validateConfig(ConfigurationLoader configurationLoader) {
                     }
 
                     if (apiType == APIType.PTERODACTYL) {
-                        if (value instanceof String uuid) {
+                        String uuid = null;
+                        if (value instanceof String) {
+                            uuid = (String) value;
+                        } else if (value instanceof Map serverConfig) {
+                            Object identifier = serverConfig.get("identifier");
+                            if (identifier instanceof String) {
+                                uuid = (String) identifier;
+                            } else {
+                                logger.warn("'servers.{}.identifier' is missing or not a string.", key);
+                                isValid = false;
+                            }
+                        } else {
+                            logger.warn("The server '{}' entry must be a string or map when type is 'pterodactyl'.", key);
+                            isValid = false;
+                        }
+                        if (uuid != null) {
```

Comment:
We probably want to add a new check that the new `whitelist` setting is a boolean and not some random value. This setting is optional, so if not provided, no error should be logged.